### ISSUE TITLE
Avoid multiple bounding boxes for a single object

### ISSUE THREAD

**Issue by jesusatgar** [AlexeyAB/darknet#4198]
Hello all,
I am having a little bit of trouble when optimizing the size of the detection bounding boxes. 
Let's say for a given object (the scratch in this metal plate) I should get one bounding box.
![scratches_31](https://user-images.githubusercontent.com/15983798/67933042-52ee7880-fbc5-11e9-968a-0cdd2266f0e0.jpg)
Desired output:
![scratches_31b](https://user-images.githubusercontent.com/15983798/67935007-4cfa9680-fbc9-11e9-9c87-5ea7e2a52386.jpg)

But instead of that, I get this result:
![scratches_31a](https://user-images.githubusercontent.com/15983798/67935035-597eef00-fbc9-11e9-9a66-a20781ef0d97.jpg)

What should I do? I guess I could train on more images, but my dataset is very limited (NEU Surface Defect Database) and intra-class variations are a problem. Is there any parameter in the .cfg or in the darknet source code that I could modify in order to extend or scale the bounding boxes? 

Thanks!

**Comment by AlexeyAB** [comment]
* What cfg-file do you use?
* Check that you have desired labels in the training dataset
* Try to train this cfg file: https://github.com/AlexeyAB/darknet/files/3795047/yolo_v3_tiny_pan3_matrix_gaussian_aa_ae_mixup.cfg.txt by using the latest version of this repository

`./darknet detector train data/obj.data yolo_v3_tiny_pan3_matrix_gaussian_aa_ae_mixup.cfg yolov3-tiny.conv.14 -map`

----

yolov3-tiny.conv.14 that you can get from https://pjreddie.com/media/files/yolov3-tiny.weights by using command ./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny.conv.14 14

**Comment by arnoldhoward** [comment]
How to change the yolo_v3_tiny_pan3_matrix_gaussian_aa_ae_mixup.cfg.txt ?
Is it same as yolo_v3_tiny.cfg ?

**Comment by AlexeyAB** [comment]
@arnoldhoward Read there: https://github.com/AlexeyAB/darknet/issues/4190#issuecomment-547866109

**Comment by arnoldhoward** [comment]
Thank you for you reply.
And one more thing, can i use this model in OpenCV dnn ?

**Comment by AlexeyAB** [comment]
@arnoldhoward No.

**Comment by scipioni** [comment]
No, Gaussian_yolo is not implemented in opencv[dnn] in current master github

**Comment by jesusatgar** [comment]
> * What mAP do you get?
> * What cfg-file do you use?
> * Check that you have desired labels in the training dataset
> * Try to train this cfg file: https://github.com/AlexeyAB/darknet/files/3795047/yolo_v3_tiny_pan3_matrix_gaussian_aa_ae_mixup.cfg.txt by using the latest version of this repository
> 
> `./darknet detector train data/obj.data yolo_v3_tiny_pan3_matrix_gaussian_aa_ae_mixup.cfg yolov3-tiny.conv.14 -map`
> 
> yolov3-tiny.conv.14 that you can get from https://pjreddie.com/media/files/yolov3-tiny.weights by using command ./darknet partial cfg/yolov3-tiny.cfg yolov3-tiny.weights yolov3-tiny.conv.14 14

Hi,

I'm using yolov3-tiny with ~75% mAP. The dataset is correctly labeled.

Will train using that .cfg, thanks!
